src/torrent_info.cpp

```diff
@@ -1481,6 +1481,13 @@ namespace {
 	{
 		TORRENT_ASSERT_PRECOND(f >= file_index_t(0));
 		if (f >= m_piece_layers.end_index()) return {};
+		if (m_files.pad_file_at(f)) return {};
+
+		if (m_files.file_size(f) <= piece_length()) {
```

Comment:
I know it's not the only one in this file, trailing `{` in the same line :)

Reply:
oops :)